core/src/main/java/com/nerds/gamejam/ecs/component/AnimationComponent.java

```diff
@@ -7,15 +7,26 @@
 
 public class AnimationComponent extends Component{
 
-    public Animation<TextureRegion> animation;
-    public float elapsedTime;
+    private String animationReference;
+    private float elapsedTime;
 
     public AnimationComponent() {
         //keep artemis happy
     }
 
-    public AnimationComponent(Array<TextureRegion> frames, float animationRuntime, Animation.PlayMode playMode) {
-        this.animation = new Animation<>(animationRuntime, frames, playMode);
+    public AnimationComponent(String animationReference) {
+        this.animationReference = animationReference;
     }
 
+    public String getAnimationReference() {
```

Comment:
No need to have the getters/setters, just make the fields public, components shouldnt do any work on their fields

Reply:
This legitimately hurts my soul 😭 public fields are the Devils work!